app/routers/chargesfiles.py

```diff
@@ -10,9 +13,15 @@
     "",
     response_model=LimitOffsetPage[ChargesFileRead],
 )
-async def get_charges_files():
-    # pending implementation
-    pass  # pragma: no cover
+async def get_charges_files(
```

Comment:
We must force the logic so affiliates can only see charges file they own like we do with systems https://github.com/softwareone-platform/ffc-finops-operations/blob/main/app/routers/systems.py#L21.